vendor/k8s.io/kubernetes/cmd/kubelet/app/bootstrap.go

```diff
@@ -42,7 +42,7 @@ const (
 // The kubeconfig at bootstrapPath is used to request a client certificate from the API server.
 // On success, a kubeconfig file referencing the generated key and obtained certificate is written to kubeconfigPath.
 // The certificate and key file are stored in certDir.
-func bootstrapClientCert(kubeconfigPath string, bootstrapPath string, certDir string, nodeName types.NodeName) error {
+func BootstrapClientCert(kubeconfigPath string, bootstrapPath string, certDir string, nodeName types.NodeName) error {
```

Comment:
I was on the fence about whether I wanted to move the code for this in 1.8 or not to a different package.